mapie/metrics/regression.py

```diff
@@ -476,15 +476,15 @@ def coverage_width_based(
     >>> y_preds_up = np.array([6, 9, 10, 12.5, 12])
     >>> eta = 0.01
     >>> alpha = 0.1
```

Comment:
We can use confidence_level instead of alpha in the example (ie `confidence_level = 0.9`[...])